python/ray/actor.py

```diff
@@ -271,7 +272,21 @@ def _bind(
             options,
             other_args_to_resolve=other_args_to_resolve,
         )
-        return node
+
+        if node.num_returns > 1:
+            output_nodes: List[ClassMethodNode] = []
+            for i in range(node.num_returns):
+                output_node = ClassMethodNode(
+                    f"return_idx_{i}",
+                    (node, i),
+                    dict(),
+                    dict(),
+                    {IS_CLASS_METHOD_OUTPUT_KEY: True},
+                )
```

Comment:
I haven't thought too much about the pros and cons. But would it be cleaner if we introduce a new node type as opposed to reusing `ClassMethodNode`?

Reply:
I think both ways are valid (whether introducing a new node type or not). @stephanie-wang Stephanie suggests this way in the current stage.